packages/design-system/src/styles/components/_Table.scss

```diff
@@ -4,7 +4,7 @@
   border-collapse: collapse;
   border-spacing: 0;
   margin: 0;
-  text-align: left;
+  text-align: start;
```

Comment:
The missing left alignment in the VRT diffs appears to be coming from `th` user agent styles taking precedence over this assignment. My browser has the following user agent styles:

```css
th {
    display: table-cell;
    vertical-align: inherit;
    font-weight: bold;
    text-align: -internal-center;
}
```